src/components/Directory/MentorCard/MentorCard.js

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import useStyles from './ MentorCard.styles';
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+import Divider from '@material-ui/core/Divider';
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+
+const MentorCard = ({
+  firstName,
+  lastName,
+  title,
+  mentorSkills,
+  bio,
+  initials,
+  onTestClick,
+}) => {
+  // useEffect(() => {
+  //   alert('making fetch Request');
+  // });
+
+  const isMobile = useMediaQuery('(max-width:767px)');
+  const classes = useStyles({ isMobile });
+  const {
+    root,
+    avatar,
+    booking,
+    skillList,
+    jobField,
+    mentorNameField,
+    bioSection,
+  } = classes;
+
+  const handleClick = (e) => {
+    e.preventDefault();
+    if (onTestClick) return onTestClick();
+    // open modal for scheduling;
+    alert('BOOKING WILL BE SCHEDULED');
+  };
+  return (
+    <Card className={root}>
+      <CardHeader
+        avatar={
+          <Avatar aria-label="Mentor Avatar" className={avatar}>
+            {initials}
+          </Avatar>
+        }
+        action={
+          <Button
+            aria-label="Booking"
+            className={booking}
+            variant="outlined"
+            onClick={handleClick}
+            role="button"
+          >
+            BOOK ME
+          </Button>
+        }
+        title={
+          <Typography
+            variant="body1"
+            className={mentorNameField}
+            data-testid="mentorNameField"
+          >
+            {firstName} {lastName}
+          </Typography>
+        }
+        subheader={
+          <Typography
+            variant="caption"
+            className={jobField}
+            data-testid="jobTitleField"
+          >
+            {title}
+          </Typography>
+        }
+      />
+      {isMobile ? null : <Divider light={false} />}
+      <CardContent>
+        <Typography
+          variant="body2"
+          className={skillList}
+          gutterBottom
+          data-testid="skillList"
+        >
```

Comment:
es6

Reply:
also let's think of a way of being able to map an array of Typography instead of hardcoding each one